src/main/java/frc/robot/commands/RotateTurret.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Limelight;
+import frc.robot.subsystems.Turret;
+
+public class RotateTurret extends CommandBase {
+
+    private Turret turret;
+    private Limelight lime;
+
+    public RotateTurret() {
+        addRequirements(turret, lime);
+    }
+
+    @Override
+    public void execute() {
+        lime.turnOn();
+        if (lime.canSeeTarget() && lime.getHorizontalAngle() > 1.5) {
```

Comment:
Before this code is loaded on the robot, I'd like to see code added that prevents over-rotation of the turret, as that could cause serious damage to the robot. I believe you can set this on the motor controller. 

It looks like a forward limit is set, but not a backward limit. 

That being said, this code will keep trying to rotate the turret past that point, getting stuck.